helix-term/src/ui/picker.rs

```diff
@@ -257,8 +257,37 @@ impl<T: 'static> Component for FilePicker<T> {
     }
 
     fn handle_event(&mut self, event: Event, ctx: &mut Context) -> EventResult {
```

Comment:
I think it would be good to change `ctx` to `cx` to be consistent with the rest of the codebase.